src/Kinepolis/ValueObject/ParsedMovie.php

```diff
@@ -33,7 +33,7 @@ public function __construct(
         string $externalId,
         Title $title,
         LocationId $locationId,
-        Description $description,
+        ?Description $description,
```

Comment:
Another approach is only to have the required or non-nullable parameters inside the constructor and add a method `withDescription` which does the typical cloning, setting, and returning.